### PR TITLE
Sous Chefs Adoption

### PR DESCRIPTION
**Pull request by ramereth**
Update tests and do other various cleanup.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please provide a summary of your Pull Request.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: attributes/vc14.rb [hunk 1]
```diff
@@ -62,11 +62,11 @@
 default['vcruntime']['vc14']['x64']['14.14.26429.4']['name']      = 'Microsoft Visual C++ 2017 Redistributable (x64) - 14.14.26429'
 
 # Microsoft Visual C++ 2019 Redistributable
```
Comment (derekgroh) [inline attributes/vc14.rb:64]: The intent is to leave all prior versions if someone wants to use that build but default to the latest version in the cookbook.

**Comment by derekgroh** [comment]
Sourced the issues source, GHActions windows-latest comes packaged with:
- Microsoft Visual C++ 2019 X86 Debug Runtime - 14.28.29334   
- Microsoft Visual C++ 2019 X86 Minimum Runtime - 14.28.29334                            
- Microsoft Visual C++ 2019 X86 Additional Runtime - 14.28.29334
- Microsoft Visual C++ 2019 X64 Minimum Runtime - 14.28.29334                            
- Microsoft Visual C++ 2019 X64 Debug Runtime - 14.28.29334                              
- Microsoft Visual C++ 2019 X64 Additional Runtime - 14.28.29334                         
which is newer than what is provided by the source url, which will cause vc14 to fail.

The fix would be to accept error code `1638`, however this highlights that when Microsoft changes the package version of the url this resource will fail as the hash will not longer be correct. I'm not aware of a method to work around this, do you have any ideas?

**Review by Xorima — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [2.1.0](https://github.com/sous-chefs/vcruntime/releases/tag/2.1.0)

**Pull request merged**